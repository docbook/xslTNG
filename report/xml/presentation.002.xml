<book xmlns="http://docbook.org/ns/docbook"
      xmlns:xlink="http://www.w3.org/1999/xlink"
      xmlns:xi="http://www.w3.org/2001/XInclude"
      version="5.1">
<info>
<title>Unit test: presentation.002</title>
<author><personname>Norm Tovey-Walsh</personname></author>
<pubdate>2023-06-06</pubdate>
<meta xmlns="http://www.w3.org/1999/xhtml"
      name="localStorage.key" content="windowLocation"/>
<annotation xml:id="a1"><para>Annotations are interesting for the same reason.</para></annotation>
<annotation xml:id="a2"><para>Press <keycap>S</keycap>.</para></annotation>
</info>

<article>
<title>Footnotes and annotations</title>
<para>Presentation mode<footnote xml:id="f1"><para>Better documented in the guide and
in the presentation.001 unit test.</para></footnote> implements paged navigation through a document.
As the name implies, it’s designed for use in presentations:</para>
<itemizedlist>
<listitem><para>Footnotes are <phrase annotations="a1">interesting</phrase> because they need to appear on each
page.</para></listitem>
<listitem><para>For the moment, the cross-linking between the inline
reference and the bottom-of-the-page footnote is removed.<footnote>
<para>On small pages, that shouldn’t be an issue.</para></footnote>
</para>
</listitem>
</itemizedlist>
</article>

<article>
<title>More footnotes and annotations</title>

<itemizedlist>
<listitem><para>Speaker notes can be placed on any page.</para></listitem>
<listitem><para>They are not displayed as part of the normal
presentation.<footnoteref linkend="f1"/></para></listitem>
<listitem><para>They can be revealed by selecting <phrase annotations="a2">notes view</phrase>.
</para></listitem>
</itemizedlist>
</article>

<article>
<title>Variable list reveal</title>

<para>This isn’t really related to footnotes and annotations,
but it needs testing.</para>

<variablelist role="reveal">
<varlistentry>
<term>First</term>
<listitem><para>The first item.</para></listitem>
</varlistentry>
<varlistentry>
<term>Second</term>
<listitem><para>The second item.</para></listitem>
</varlistentry>
<varlistentry>
<term>Third</term>
<listitem><para>The third item.</para></listitem>
</varlistentry>
</variablelist>
</article>

<article>
<title>Footnote position</title>

<para role="speaker-notes">This long list with all the lazily repeated
items is just to test that if the content runs off the bottom of the
page (i.e., it is longer than “100%”), the footnote footer still looks
reasonable.</para>

<itemizedlist>
<listitem><para>Speaker notes can be placed on any page.</para></listitem>
<listitem><para>They are not displayed as part of the normal
presentation.<footnoteref linkend="f1"/></para></listitem>
<listitem><para>They can be revealed by selecting <phrase annotations="a2">notes view</phrase>.
</para></listitem>
<listitem><para>Speaker notes can be placed on any page.</para></listitem>
<listitem><para>They are not displayed as part of the normal
presentation.<footnoteref linkend="f1"/></para></listitem>
<listitem><para>They can be revealed by selecting <phrase annotations="a2">notes view</phrase>.
</para></listitem>
<listitem><para>Speaker notes can be placed on any page.</para></listitem>
<listitem><para>They are not displayed as part of the normal
presentation.<footnoteref linkend="f1"/></para></listitem>
<listitem><para>They can be revealed by selecting <phrase annotations="a2">notes view</phrase>.
</para></listitem>
<listitem><para>Speaker notes can be placed on any page.</para></listitem>
<listitem><para>They are not displayed as part of the normal
presentation.<footnoteref linkend="f1"/></para></listitem>
<listitem><para>They can be revealed by selecting <phrase annotations="a2">notes view</phrase>.
</para></listitem>
<listitem><para>Speaker notes can be placed on any page.</para></listitem>
<listitem><para>They are not displayed as part of the normal
presentation.<footnoteref linkend="f1"/></para></listitem>
<listitem><para>They can be revealed by selecting <phrase annotations="a2">notes view</phrase>.
</para></listitem>
</itemizedlist>
</article>

</book>
