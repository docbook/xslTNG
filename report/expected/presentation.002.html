<!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" class="no-js"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/><script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script><title>Unit test: presentation.002</title><meta name="viewport" content="width=device-width, initial-scale=1.0"/><link href="https://purl.org/dc/elements/1.1/" rel="schema.dc"/><meta content="2011-04-22T17:02:00-06:00" name="dc.modified"/><meta name="dc.created" content="2023-06-06"/><meta content="DocBook xslTNG" name="generator"/><link href="./css/docbook.css" rel="stylesheet" media="screen"/><meta name="localStorage.key" content="windowLocation"/><script src="./js/annotations.js" defer="defer"></script></head><body class="home"><nav class="top"></nav><main><article class="book division"><header><h1>Unit test: presentation.002</h1><div class="author"><h3><span class="first-last personname">Norm Tovey-Walsh</span></h3></div><p class="pubdate"><time class="pubdate" datetime="2023-06-06">06 Jun 2023</time></p></header><div class="list-of-titles"><div class="lot toc"><div class="title">Table of Contents</div><ul class="toc"><li><a href="#R_article1">Footnotes and annotations</a></li><li><a href="#R_article2">More footnotes and annotations</a></li><li><a href="#R_article3">Variable list reveal</a></li><li><a href="#R_article4">Footnote position</a></li></ul></div></div><section id="R_article1" class="article component"><header><h1>Footnotes and annotations</h1></header><p>Presentation mode<sup id="f1-fref" class="footnote-number"><a href="#f1-fnote">1</a></sup> implements paged navigation through a document.
As the name implies, it’s designed for use in presentations:</p><ul class="itemizedlist"><li><p>Footnotes are <span class="phrase">interesting<a class="annomark" href="#a1"><sup>⌖</sup><sup class="num">1</sup></a></span> because they need to appear on each
page.</p></li><li><p>For the moment, the cross-linking between the inline
reference and the bottom-of-the-page footnote is removed.<sup id="R_article1_itemizedlist1_listitem2_para1_footnote1-fref" class="footnote-number"><a href="#R_article1_itemizedlist1_listitem2_para1_footnote1-fnote">2</a></sup>
</p></li></ul></section><section id="R_article2" class="article component"><header><h1>More footnotes and annotations</h1></header><ul class="itemizedlist"><li><p>Speaker notes can be placed on any page.</p></li><li><p>They are not displayed as part of the normal
presentation.<sup id="R_article2_itemizedlist1_listitem2_para1_footnoteref1-fref" class="footnote-number"><a href="#f1-fnote">1</a></sup></p></li><li><p>They can be revealed by selecting <span class="phrase">notes view<a class="annomark" href="#a2"><sup>⌖</sup><sup class="num">2</sup></a></span>.
</p></li></ul></section><section id="R_article3" class="article component"><header><h1>Variable list reveal</h1></header><p>This isn’t really related to footnotes and annotations,
but it needs testing.</p><dl class="reveal variablelist"><div class="varlistentry"><dt><span class="term">First</span></dt><dd><p>The first item.</p></dd></div><div class="varlistentry"><dt><span class="term">Second</span></dt><dd><p>The second item.</p></dd></div><div class="varlistentry"><dt><span class="term">Third</span></dt><dd><p>The third item.</p></dd></div></dl></section><section id="R_article4" class="article component"><header><h1>Footnote position</h1></header><p class="speaker-notes">This long list with all the lazily repeated
items is just to test that if the content runs off the bottom of the
page (i.e., it is longer than “100%”), the footnote footer still looks
reasonable.</p><ul class="itemizedlist"><li><p>Speaker notes can be placed on any page.</p></li><li><p>They are not displayed as part of the normal
presentation.<sup id="R_article4_itemizedlist1_listitem2_para1_footnoteref1-fref" class="footnote-number"><a href="#f1-fnote">1</a></sup></p></li><li><p>They can be revealed by selecting <span class="phrase">notes view<a class="annomark" href="#a2"><sup>⌖</sup><sup class="num">2</sup></a></span>.
</p></li><li><p>Speaker notes can be placed on any page.</p></li><li><p>They are not displayed as part of the normal
presentation.<sup id="R_article4_itemizedlist1_listitem5_para1_footnoteref1-fref" class="footnote-number"><a href="#f1-fnote">1</a></sup></p></li><li><p>They can be revealed by selecting <span class="phrase">notes view<a class="annomark" href="#a2"><sup>⌖</sup><sup class="num">2</sup></a></span>.
</p></li><li><p>Speaker notes can be placed on any page.</p></li><li><p>They are not displayed as part of the normal
presentation.<sup id="R_article4_itemizedlist1_listitem8_para1_footnoteref1-fref" class="footnote-number"><a href="#f1-fnote">1</a></sup></p></li><li><p>They can be revealed by selecting <span class="phrase">notes view<a class="annomark" href="#a2"><sup>⌖</sup><sup class="num">2</sup></a></span>.
</p></li><li><p>Speaker notes can be placed on any page.</p></li><li><p>They are not displayed as part of the normal
presentation.<sup id="R_article4_itemizedlist1_listitem11_para1_footnoteref1-fref" class="footnote-number"><a href="#f1-fnote">1</a></sup></p></li><li><p>They can be revealed by selecting <span class="phrase">notes view<a class="annomark" href="#a2"><sup>⌖</sup><sup class="num">2</sup></a></span>.
</p></li><li><p>Speaker notes can be placed on any page.</p></li><li><p>They are not displayed as part of the normal
presentation.<sup id="R_article4_itemizedlist1_listitem14_para1_footnoteref1-fref" class="footnote-number"><a href="#f1-fnote">1</a></sup></p></li><li><p>They can be revealed by selecting <span class="phrase">notes view<a class="annomark" href="#a2"><sup>⌖</sup><sup class="num">2</sup></a></span>.
</p></li></ul></section></article><footer><div class="footnotes"><hr/><div class="footnote" id="f1-fnote"><div class="footnote-number"><sup class="footnote-number"><a href="#f1-fref">1</a></sup></div><div class="footnote-body"><p>Better documented in the guide and
in the presentation.001 unit test.</p></div></div><div class="footnote" id="R_article1_itemizedlist1_listitem2_para1_footnote1-fnote"><div class="footnote-number"><sup class="footnote-number"><a href="#R_article1_itemizedlist1_listitem2_para1_footnote1-fref">2</a></sup></div><div class="footnote-body"><p>On small pages, that shouldn’t be an issue.</p></div></div></div><div class="annotations"><div class="annotation-wrapper title">Annotations</div><div id="a1" class="annotation-wrapper"><div class="annotation-body"><div class="annotation-header"><div class="annotation-close"></div><div class="annotation-title"><span class="annomark"><sup>⌖</sup><sup class="num">1</sup> </span></div></div><div class="annotation-content"><p>Annotations are interesting for the same reason.</p></div></div></div><div id="a2" class="annotation-wrapper"><div class="annotation-body"><div class="annotation-header"><div class="annotation-close"></div><div class="annotation-title"><span class="annomark"><sup>⌖</sup><sup class="num">2</sup> </span></div></div><div class="annotation-content"><p>Press <span class="keycap">S</span>.</p></div></div></div></div></footer></main><nav class="bottom"></nav><script type="text/html" class="annotation-close"><span>╳</span></script></body></html>